### DISCUSSION TITLE
Tree scroll

### DISCUSSION THREAD

**Discussion by giladvidal** [UI5/webcomponents-react#3932] [Q&A] [answered]
Hello,
I have a Tree component where each TreeItem has an id. So, when building the tree I can have an item selected and expanded - works fine. 

However, I'd like to also scroll to that point. Trying document.getElementById('item100').scrollIntoView() does not seem to work. Is there a limitation with the Tree component? 

Thanks,
Gilad

**Comment by Lukas742** [comment, marked as answer]
Hi @giladvidal 

`scollIntoView` seems to work fine for me. --> https://codesandbox.io/s/affectionate-wave-fhyune?file=/src/App.js

**Comment by giladvidal** [comment]
Hi @Lukas742 , when you are around all is working :)
Maybe the fact that it is in a splitted layout causes a problem? I'll check again and update.
Thanks.

**Comment by giladvidal** [comment]
Hi @Lukas742 , after upgrading to the libraries in your example (list below) it worked. Any issue with previous versions related to this topic?

"@ui5/webcomponents": "~1.9.3",
"@ui5/webcomponents-fiori": "~1.9.3",
"@ui5/webcomponents-react": "~1.7.0",

**Reply by Lukas742** [reply]
The Tree is developed by the UI5 Web Components team and I couldn't find anything about it in their [release log](https://github.com/SAP/ui5-webcomponents/releases). So, if you want to know when this was changed, it's probably best to ask them in their repo.
